inf-clojure.el

```diff
@@ -1254,15 +1254,43 @@ See variable `inf-clojure-buffer'."
   "Return DATA if and only if it is a list."
   (when (listp data) data))
 
-(defun inf-clojure-completions (expr)
-  "Return a list of completions for the Clojure expression starting with EXPR."
+(defun inf-clojure-list-completions (expr)
+  "Return completions for the Clojure expression starting with EXPR.
+
+Its only ability is to parse a list of candidate strings, every
+other EXPR will be discarded and nil will be returned."
```

Comment:
Isn't EXPR only the completion prefix? Probably you didn't mean to reference it here. On a related note - we might pick a better name for this param.

Reply:
Open to suggestions :)

Reply:
Will refactor